### ISSUE TITLE
UX for Settings/Text size

### ISSUE THREAD

**Issue by payeisa** [StatCan/StatcanPublicationUAT#125]
Is there a way that we can a consistent design/color scheme for the Text size screen? (i.e. instead of having the green slider and the blue line, can we use the same color scheme as what is used throughout the app?)

![image](https://user-images.githubusercontent.com/89025704/146005793-19f6ec39-702c-409c-960c-6c5af5ff0dce.png)

**Comment by ForumG-SC** [comment]
@payeisa 
![image](https://user-images.githubusercontent.com/84411852/146028508-79b36d4c-68f9-44d0-9ba0-7834772e9a1a.png)

Does this works ?

For apple , if we have white circle in the slider it is visible , however in android if we keep white circle in the slider it is not visible so I changed it to grey, however if you want me to change it to a specific color , please let me know.

**Comment by payeisa** [comment]
@ForumG-SC I would recommend discussing with Rajat, as he would be in a better position to provide feedback based on design and accessibility,

Just to add, can we also make the "Bold text" slider a different color? Other than green?

**Comment by ForumG-SC** [comment]
> @ForumG-SC I would recommend discussing with Rajat, as he would be in a better position to provide feedback based on design and accessibility,
> 
> Just to add, can we also make the "Bold text" slider a different color? Other than green?

Yes, we can change the bold text bottom to different color than green.

**Comment by ForumG-SC** [comment]
@Rajat-Sen Please provide with color selection for the slider and the bold text button, thanks.

**Comment by ForumG-SC** [comment]
@Rajat-Sen Please provide with color selection for the slider and the bold text button, thanks.

**Comment by Rajat-Sen** [comment]
@payeisa Would it be okay for the MVP to use system colors. To exemplify, in iOS; it is matching the system. However, on the android; the commonly used color is dark grey and can we stick with it for the time being. After the launch, we can improve it. And, we can leave this ticket open under enhancement after changing the color to dark grey!

**Comment by payeisa** [comment]
@Rajat-Sen Noted, and I agree with using the system colors for MVP. Thanks, Isabelle

**Comment by Rajat-Sen** [comment]
Hi Forum, 

For the MVP, let it only affect the Level-2 screen.

After the MVP, it should affect all the elements on all screens.

**Comment by Rajat-Sen** [comment]
We have been receiving inquires regarding this feature not working properly.

**Comment by ForumG-SC** [comment]
@Rajat-Sen Hi Rajat, this ticket was opened for design/color scheme which had been discussed and implemented. Please create separate ticket for the feature's functionality. Thanks.
